radius.py

```diff
@@ -486,6 +486,8 @@ def handle_register(rocket):
 def handle_cgit(rocket):
     cgit_env = os.environ.copy()
     cgit_env['PATH_INFO'] = rocket.path_info.removeprefix('/cgit')
+    if (re.search("^/[\w\-\.]+/plain(/|$)", cgit_env['PATH_INFO'])):
```

Comment:
I don't love this regex. Hard to read (imo) and I can't help but worry about edge cases (what if you had a repo named `plain` or something stupid). I think maybe just hiding the button is enough. Until we actually fix the plain feature, if you go to a url that isn't linked anywhere and it crashes I think that that is acceptable (especially given that we handle those errors more elegantly in the follow up commit).